docs/design.md

```diff
@@ -94,7 +94,8 @@ uaid string REQUIRED
 
 channelIDs list of strings REQUIRED
```

Comment:
I think we can just remove this entirely, unless it's meant to document the protocol instead of our implementation. The FxOS Simple Push client doesn't pass `channelIDs`, either.